config/opal_setup_wrappers.m4

```diff
@@ -231,6 +231,7 @@ AC_DEFUN([OPAL_SETUP_RUNPATH],[
                     AC_MSG_RESULT([yes (-Wl,--enable-new-dtags)])],
                    [AC_MSG_RESULT([no])])
     AC_LANG_POP([C])
+m4_ifdef([project_ompi],[
```

Comment:
I understand that this was done to make this commit be a minimal/trivial commit.  But effectively putting a new block in the middle of an existing block without re-indenting everything is... misleading, at best.  If this section is going to be conditionally used, it should probably be noted better somehow -- perhaps by becoming its own M4 macro.